artichoke-backend/src/extn/core/kernel/trampoline.rs

```diff
@@ -54,6 +54,20 @@ pub fn puts(interp: &mut Artichoke, args: Vec<Value>) -> Result<Value, Exception
     Ok(interp.convert(None::<Value>))
 }
 
+pub fn p(interp: &mut Artichoke, args: Vec<Value>) -> Result<Value, Exception> {
+    for value in &args {
+        let display = value.inspect(interp);
+        let mut borrow = interp.0.borrow_mut();
+        borrow.output.puts(display);
```

Comment:
unfortunately the current implementation of `output` doesn't allow this to be tested easily. There is an in-memory `output` backend, but it requires a compile time flag to flip it, which we can't change in tests.

If you can validate the output is written to stdout in `airb` or with `artichoke -e 'p 1'`, that's good enough for me.

Reply:
Is there a way to execute either `airb` or `artichoke` as part of the normal `cargo` test suite? I was looking into how `ripgrep` executed their tests but I wasn't sure if there was something pre-existing that I haven't been able to find. It's also pretty heavyweight so didn't want to introduce a lot boilerplate and complexity if not needed.

Reply:
Oh I didn’t mean in the test suite. I meant one off manually for now.